### PR TITLE
Monitor support for Image Verification

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Monitor has been modified to add support for Image verification

### CODE REVIEW

File: binary_transparency/firmware/cmd/ft_monitor/main.go [hunk 1]
```diff
@@ -94,6 +97,32 @@ func main() {
 			}
 
 			glog.V(1).Infof("Inclusion proof for leafhash 0x%x verified", lh)
+
+			statement := manifest.Value
+			stmt := api.FirmwareStatement{}
+			if err := json.NewDecoder(bytes.NewReader(statement)).Decode(&stmt); err != nil {
+				glog.Warningf("Fimware Statement decoding from manifest Failed reason %q", err)
```
Comment (AlCutter) [inline binary_transparency/firmware/cmd/ft_monitor/main.go:104]: typo: firmware

Comment (AlCutter) [reply binary_transparency/firmware/cmd/ft_monitor/main.go:104]: (also can you lowercase "Failed" while you're here?)

File: binary_transparency/firmware/cmd/ft_monitor/main.go [hunk 2]
```diff
@@ -94,6 +97,32 @@ func main() {
...
+				glog.Warningf("Unable to decode FW Metadata from Statement %q", err)
+				continue
+			}
+			// Fetch the Image from FT Personality
+			image, err := c.GetFirmwareImage(meta.FirmwareImageSHA512)
+			if err != nil {
+				glog.Warningf("Unable to GetFirmwareImage for Firmware with Hash 0x%x , reason %q", meta.FirmwareImageSHA512, err)
+				continue
+			}
+			// Verify Image Hash from log Manifest match the actual image hash
```
Comment (AlCutter) [inline binary_transparency/firmware/cmd/ft_monitor/main.go:119]: match -> matches

**Review by AlCutter — APPROVED** [review]
Nice one - just a couple of tiny nits/typos

**Review by yogeshbdeshpande — COMMENTED** [review]
Incorporated code review comments!

File: binary_transparency/firmware/cmd/ft_monitor/main.go (hunk 1 above)
Comment (yogeshbdeshpande) [reply binary_transparency/firmware/cmd/ft_monitor/main.go]: Done

**Pull request merged**